src/openapi/peridio-admin-openapi.yaml

```diff
@@ -4645,6 +4645,8 @@ components:
           $ref: "#/components/schemas/binary-signature-signature"
         signing_key_prn:
           $ref: "#/components/schemas/prn"
+        public_value:
+          $ref: "#/components/schemas/signing-key-value"
```

Comment:
We should add a description that says its the PEM encoding.

When you do that you need to make sure that other spots don't get that description when they shouldn't (if its not actually PEM there). May need two signing-key-value-pem and signing-key-value-raw